indico/web/client/styles/partials/_boxes.scss

```diff
@@ -217,7 +217,6 @@ $i-box-padding: 10px;
     justify-content: space-between;
     align-items: center;
     background-color: $light-gray;
-    overflow: hidden;
```

Comment:
I think removing this is dangerous and has a good chance for breaking the layout somewhere. I'd rather do it in local CSS when needed there..